### ISSUE TITLE
StudentProfilePageUiTest test fails occasionally on line 106

### ISSUE THREAD

**Issue by MrFool** [TEAMMATES/repo#3571]
Expecting a different status message than was shown

![screen shot 2015-05-30 at 8 33 24 pm](https://cloud.githubusercontent.com/assets/6291947/7897343/25fdcb90-070b-11e5-8f60-a0e9087d8925.png)

**Comment by MrFool** [comment]
Digged a little into this, it seems that our current test code doing this ->

![screen shot 2015-06-04 at 10 36 49 am](https://cloud.githubusercontent.com/assets/6291947/7975850/a4c8ec46-0aa5-11e5-963a-d81c484ed3cb.png)
![screen shot 2015-06-04 at 10 36 41 am](https://cloud.githubusercontent.com/assets/6291947/7975849/a4c5ee7e-0aa5-11e5-9b95-5b25b09c6816.png)

is not always consistent because Selenium .click MAY or MAY NOT trigger a wait for page to load, that means that our waitForPageToLoad("10000") MAY or MAY NOT trigger causing the preceding line ->

verification of status to run and because "$$.short.name is not acceptable..." is what was shown before the next page is loaded, the test fails.

First suspicion rose when I looked at Selenium .click()'s documentation, and this stack over flow thread talks about the same issue.

http://stackoverflow.com/questions/15122864/selenium-wait-until-document-is-ready

One of the proposed "workaround", because it is in no way Selenium's fault for not being able to detect it, is to compound implicit waits with explicit waits to reduce the chances of this test not being able to pass. I'm gonna open a PR soon and add a 1 second explicit wait and we can see if it continues to happen regularly or whether the rate of passing / failing has been changed.
